### ISSUE TITLE
bug: Java client query serialization issue with Extended Scalar DateTime

### ISSUE THREAD

**Issue by amitgupta1202** [Netflix/dgs-framework#290]
## Expected behavior

We are using extended Scalar DateTime as an input parameter. 

GraphQL Schema

        scalar DateTime

        type Query {
            exampleQuery(someDateTime: DateTime!): [String]!
        }

Implementation:

```
@DgsComponent
public class ScalarConfig {
    @DgsRuntimeWiring
    public RuntimeWiring.Builder addScalar(RuntimeWiring.Builder builder) {
        return builder.scalar(ExtendedScalars.DateTime);
    }
}

@DgsComponent
public class SupplierQueryResolver {

    @DgsQuery
    public String exampleQuery(@InputArgument(value = "someDateTime") OffsetDateTime someDateTime) {
        return "hello";
    }
}
```

We  make a call using generated Java client and expect response back as "hello".

## Actual behavior
When we execute the query using the generated Java Client we get error 
`com.netflix.graphql.dgs.exceptions.QueryException: Invalid Syntax : Invalid syntax: token recognition error at: '-01' at line 1 column 39`

## Steps to reproduce
```
@SpringBootTest(webEnvironment = RANDOM_PORT)
public class ExampleExtendedScalarDateTime {

    @Autowired
    private DgsQueryExecutor dgsQueryExecutor;

    @Test
    void extended_scalar_DateTime_serialization_issue() {
        var epochDateTime = OffsetDateTime.ofInstant(Instant.EPOCH, ZoneId.from(ZoneOffset.UTC));
        var query = ExampleQueryGraphQLQuery.newRequest().someDateTime(epochDateTime).build();
        var request = new GraphQLQueryRequest(query);

        var jsonResult = dgsQueryExecutor.executeAndGetDocumentContext(request.serialize()).jsonString();
        assertThat(jsonResult).isEqualTo("{\"data\":{\"exampleQuery\":\"hello\"}}");
    }
}
```

Above test fails with exception described above.

**Comment by amitgupta1202** [comment]
The issue seems to be in the Code GraphQLQueryRequest:55, only String is serialised by wrapping value double quote, where are rest of value are not, which in case of Extended Scalar Types is definitely not true. Types Such as OffsetDateTime, OffSetTime, LocalDate, Locale will produce a String value.

**Comment by amitgupta1202** [comment]
I have got a temporary fix for it as patch file https://gist.github.com/amitgupta1202/c0649cc41395fc4cda12912a0832b871

fix might for be the best but testcases explains the issue and might be useful.

**Comment by amitgupta1202** [comment]
Pull request raised. https://github.com/Netflix/dgs-framework/pull/292

**Comment by paulbakker** [comment]
Fixed by #333
